packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx

```diff
@@ -200,6 +280,14 @@ export const CommentsItem = ({
   const [replyFormIsOpen, setReplyFormIsOpen] = useState(false);
   const [showEditState, setShowEditState] = useState(false);
   const [showParentState, setShowParentState] = useState(showParentDefault);
+  const [activePotentialComment, setActivePotentialComment] = React.useState<0|1|2>(0)
+  const potentialCommentBodies = seededShuffle([...(comment.doppelComments ?? []).slice(0, 2), comment], comment._id)
```

Comment:
`seededShuffle([...(comment.doppelComments ?? []).slice(0, 2), comment], comment._id)` is kind of annoying to read.  Can you extract the thing being shuffled to a variable?